docs/contributing/testing/load.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Load Tests
+
+Load tests are available as [k6](https://k6.io/) scripts for executing locally or in the cloud,
+focused on exercising the platform at scale.
+
+## Configuration and parameters
+
+Two environment variables are used for all tests:
+
+- `IDENTITY_URL`: URL of the
+  [Identity](https://github.com/bitwarden/server/tree/master/src/Identity) instance for
+  authentication.
+- `API_URL`: URL of the [API](https://github.com/bitwarden/server/tree/master/src/Api) instance for
```

Comment:
It sounds helpful to me to put the value of [this header](https://github.com/bitwarden/devops/blob/561cc5597f7b809e4fa1c028a022d94a45a555a7/SRE/k6-loadtests/groups.js#L40) in `appsettings.Development.json` of each service so that these can more easily be ran locally but we will then need to make it configurable with an environment variable for all other environments. Thoughts?

Reply:
@mimartin12 are we just setting `ClientWhitelist` as an environment variable? This seems built-in.

I'll submit a PR for the scripts that makes this configurable there and accepted as an environment variable, then update this doc.

Reply:
https://github.com/bitwarden/devops/pull/944